ext/drx/drx.h

```diff
@@ -459,6 +459,40 @@ void
 drx_buf_insert_buf_memcpy(void *drcontext, drx_buf_t *buf, instrlist_t *ilist,
                           instr_t *where, reg_id_t dst, reg_id_t src, ushort len);
 
+DR_EXPORT
+/**
+ * Expands AVX2 gather and AVX-512 gather and scatter instructions to a sequence of
+ * equivalent scalar load and stores, mask register bit tests, and mask register bit
+ * updates.
+ *
+ * WARNING: This function is not fully supported yet. Do not use.
+ *
+ * WARNING: The added multi-instruction loop contains several control-transfer
+ * instructions and is not straight-line code, which can complicate subsequent analysis
+ * routines.
+ *
+ * WARNING: This function makes a call to drreg_init() and drreg_exit(), extending the
```

Comment:
nit: This seems more like an informational note, not a warning?  `\note`.  Also, not sure it should include drreg_exit like that: makes it sound like it calls exit every time.

Reply:
I decided to take the note out completely, it will probably confuse the user more than it helps, thinking that there is something to do here.